includes/test/functions.php

```diff
@@ -49,6 +32,26 @@ public function test_data_iterator_malformed() {
         $output = data_iterator($string);
         $this->assertEquals($output, 'Input is not an array');
     }
+
+    /**
+     * will mark as skipped because function cannot be unit tested
+     */
+    public function test_database_connect() {
+        $this->markTestSkipped(
+            'The MySQLI extension is not available.'
+            //Requires database connection so cannot be unit tested.
```

Comment:
I would replace the text in quotes with what you have in the comment, so that the database has stored the reason that you skipped it. That way, at a later date, somebody inspecting it would be given the string (they might not necessarily be looking at the test file, but rather at the test feedback in terminal, so wouldn't necessarily see your comment). I hope that makes sense, it's a bit long-winded!